jwql/website/apps/jwql/views.py

```diff
@@ -319,7 +317,7 @@ def archived_proposals_ajax(request, inst):
             all_proposal_info['proposals'].append(proposal)
             all_proposal_info['thumbnail_paths'].append(proposal_info['thumbnail_paths'][0])
             all_proposal_info['num_files'].append(num_files)
-            #all_proposal_info['num_files'].append(proposal_info['num_files'][0])
+            # all_proposal_info['num_files'].append(proposal_info['num_files'][0])
```

Comment:
Can we remove this line?

Reply:
not sure why it was initially in there, if it was as a note or whatnot, but yes, we shouldn't leave commented code in. unless there is a comment explaining why it was intentionally left in

Reply:
Thank you!